apps/site/pages/components/organisms/filter-slider.mdx

```diff
@@ -0,0 +1,419 @@
+---
+description: The mobile `Filter` view that is rendered inside a `SlideOver` component.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/PaymentMethods___b0f344b254360daa53406cd631887adf.png
+---
+
+import { Tab, Tabs } from 'nextra-theme-docs'
+import PropsSection from 'site/components/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
+import { OverviewSection } from 'site/components/OverviewSection'
+import { SectionItem, SectionList } from 'site/components/SectionItem'
+import FilterUsage from 'site/components/Filter/FilterUsage'
+import { Button, Toast, useUI, UIProvider, Filter } from '@faststore/ui'
+import { SearchProvider } from '@faststore/sdk'
+import path from 'path'
+import { useSSG } from 'nextra/ssg'
+import { getComponentPropsFrom } from 'site/components/utilities/propsSection'
+
+export const getStaticProps = () => {
+  const componentPath = path.resolve(__filename.replace("filter-slider", "filter"))
+  const components = [
+    'FilterSlider.tsx',
+  ]
+  const [filterSliderProps] = getComponentPropsFrom(componentPath, components)
+  return {
+    props: {
+      // We add an `ssg` field to the page props,
+      // which will be provided to the Nextra `useSSG` hook.
+      ssg: {
+        filterSliderProps,
+      },
+    },
+  }
+}
+
+export const FilterSliderPropsSection = ({ component }) => {
+  // Get the data from SSG, and render it as a component.
+  const { filterSliderProps } = useSSG()
+  return {
+    FilterSlider: <PropsSection propsList={filterSliderProps} />,
+  }[component]
+}
+
+<header>
+
+# FilterSlider
+
+The mobile `Filter` view that is rendered inside a `SlideOver` component.
+
+</header>
+
+## Overview
+
+<Tabs items={['Example', 'Code']} defaultIndex="0">
```

Comment:
Since the `Overview` section is the same as `Usage` example. We can remove this `Overview` section here. 

Can you remove this section and apply `Tabs` on `Usage` example?